site/content/blog/2020-06-04-svelte-and-typescript.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Svelte <3 TypeScript
+description: Typernetically enhanced web apps
+author: Orta Therox
+authorURL: https://twitter.com/orta
+---
+
+It's been on the TODO list for a while, and it's now happening. TypeScript with Svelte is now a first class citizen of the ecosystem.
+
+We think it'll help you handle much larger Svelte code bases regardless of whether you use TypeScript or JavaScript.
```

Comment:
Might be better if this doesn't suggest that it's only applicable to large codebases? We don't want people who use Svelte for smaller projects to feel like our focus is shifting away from them, since it isn't

```suggestion
We think it'll give you a much nicer development experience — one that also scales beautifully to larger Svelte code bases — regardless of whether you use TypeScript or JavaScript.
```